### DISCUSSION TITLE
The mosaicism notation on alleles (version with heterozygote and homozygote)

### DISCUSSION THREAD

**Discussion by domsab1** [HGVSnomenclature/hgvs-nomenclature#206] [Q&A]
Hi! 

We have a variant in the COL1A2 gene at position c.1702G>A in a mosaic pattern. We would like to document this according to the latest HGVS guidelines and have encountered an issue. We managed to create the following notation: NM_000089.4:c.1702=/G>A, but it seems that this does not account for the allele situation. How can we (if possible) express this mosaic as a heterozygote or homozygote?

**Comment by jfjlaros** [comment]
If the variant is heterozygous in one part of the cells and absent in the other part, I would probably describe this as:

    NM_000089.4:c.[1702G>A][=]/[=]

If it is homozygous in one part of the cells and absent in the other part, then I would suggest the following:

    NM_000089.4:c.[1702G>A][1702G>A]/[=]

**Reply by ifokkema** [reply]
> If the variant is heterozygous in one part of the cells and absent in the other part, I would probably describe this as:
> 
> ```
> NM_000089.4:c.[1702G>A][=]/[=]
> ```

That doesn't seem to follow any rule that I know from the HGVS nomenclature website. However, `NM_000089.4:c.[1702=/G>A];[1702=]` follows the rules of the allele and mosaic syntaxes.


> If it is homozygous in one part of the cells and absent in the other part, then I would suggest the following:
>
> ```
> NM_000089.4:c.[1702G>A][1702G>A]/[=]
> ```

That would be `NM_000089.4:c.[1702=/G>A];[1702=/G>A]`, as far as I know.
However, that is extremely unlikely. Not with chimerism, but if this is mosaicism, I think the question should be interpreted differently.

If the variant is homozygous in some cells and heterozygous in others (a more likely example of mosaicism), it would be `NM_000089.4:c.[1702G>A];[1702=/G>A]`.


Relevant pages:
https://hgvs-nomenclature.org/stable/recommendations/DNA/other/
https://hgvs-nomenclature.org/stable/recommendations/DNA/alleles/

**Reply by jfjlaros** [reply]
My reasoning is as follows. 

Chimerism can consist of mosaic sets, which consist of alleles. This forms a hierarchy in which all sorts of combinations can be made, e.g., `[...];[...]/[...];[...]//[...];[...]/[...];[...]` (I forgot the semicolons in my first reply).

When reading the supplementary material of the [paper](https://pmc.ncbi.nlm.nih.gov/articles/PMC3194197/) we wrote on this, I indeed see that we made a different choice back then.

**Reply by ifokkema** [reply]
That hierarchy certainly makes a lot of sense. I don't see any examples given on the website, and I don't really think the current mosaicism nomenclature is equipped for this, but following that hierarchy, I think the notation would be `NM_000089.4:c.[1702=];[1702=]/[1702G>A];[1702=]` for mosaic heterozygosity in an otherwise wild-type organism (wild-type considering the NM_000089.4:c.1702 location, I mean), and `NM_000089.4:c.[1702G>A];[1702=]/[1702G>A];[1702G>A]` when some cells are heterozygous and some cells are homozygous.

Notes:
- `[=]` should not be used as it suggests that the entire transcript is unchanged (see the allele page).
- In the trans examples on the allele page, wild-type alleles are written last (e.g., `LRG_199t1:c.[2376G>C];[2376=]`, while the mosaic page indicates that wild-type alleles should be written first (e.g., `LRG_199t1:c.85=/T>C`). I find this slightly confusing as it creates inconsistency, and it requires additional thought when writing up the nomenclature.
- When using the mosaic notation like this, we're making some assumptions that are currently not defined in the nomenclature. Discussion is needed within the committee to see how we'll handle this. For now, I suggest to use the notation that I mentioned earlier (where mosaicism is used within the allele syntax instead of the other way around) as it fits the current nomenclature.

**Reply by jfjlaros** [reply]
Agreed.

> - `[=]` should not be used [...]

Maybe we can discuss this normalisation nightmare separately.

**Reply by ifokkema** [reply]
> Maybe we can discuss this normalisation nightmare separately.

OK! I don't think there's a discussion open on this already - could you open one? Although I might have an idea of what issues you're running into with that, I probably don't see them all at this moment.
